data/scripts/network/gm_teleport.lua

```diff
@@ -0,0 +1,10 @@
+local handler = PacketHandler(0x73)
+
+handler.onReceive = function(player, msg)
```

Comment:
Opinions on this idiom instead of
```suggestion
function handler.onReceive(player, msg)
```
? I would like to pick one and standardize

Reply:
You're right and that's how we do with event callbacks too...

I'll update the PR. 🤓